rust-runtime/aws-smithy-json/src/deserialize/token.rs

```diff
@@ -221,14 +220,12 @@ pub fn expect_timestamp_or_null(
             .map(|v| {
                 if v.is_nan() {
                     Err(Error::new(
-                        ErrorReason::Custom(Cow::Owned("NaN is not a valid epoch".to_string())),
+                        ErrorKind::Custom(Cow::Owned("NaN is not a valid epoch".to_string())),
                         None,
                     ))
                 } else if v.is_infinite() {
                     Err(Error::new(
-                        ErrorReason::Custom(Cow::Owned(
-                            "Infinity is not a valid epoch".to_string(),
-                        )),
+                        ErrorKind::Custom(Cow::Owned("Infinity is not a valid epoch".to_string())),
```

Comment:
Is calling to_string even necessary for these tests? Why not just use `Cow::Borrowed`?

Reply:
Not sure what I was thinking. I went through and improved several of these.